broadinstitute/single_cell_portal_core
# Update metadata labels per EFO changes for 10x protocols (SCP-3371)

Like #994, this updates synthetic test data and an image to reflect upstream EFO changes.  See broadinstitute/single_cell_portal/pull/122 for related changes.

To test:
* Delete any pre-existing copy of synthetic study for human lymph node 
* `rails console`
* `SyntheticStudyPopulator.populate('human_lymph_node')`
* Wait a few minutes.
* Explore created study in SCP UI, verify it has visualizations (which entails conventional metadata).

This relates to SCP-3371.
